TrackletAlgorithm/TrackletCalculator.h

```diff
@@ -8,7 +8,9 @@
 #include "AllStubMemory.h"
 #include "TrackletParameterMemory.h"
 #include "TrackletProjectionMemory.h"
-
+#ifndef __SYNTHESIS__
+#include "SWLUTReader.h"
+#endif
```

Comment:
@jasonfan393 Since you also include this header file in TrackletProcessor_parameters.h, do you really need it here?